src/instructlab/model/serve_backend.py

```diff
@@ -23,6 +24,43 @@ def signal_handler(num_signal, __):
     sys.exit(0)
 
 
+def check_gpu_count(gpus: int, available_gpus: int) -> bool:
+    return gpus <= available_gpus
```

Comment:
Do we need a separate function for this?

Reply:
I think it's for testing mock. Has its pros and cons I guess...